release-please-config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "bootstrap-sha": "1354831baeafb52965a8b2f5f80df5c43e414c99",
+    "exclude-paths": [
+      ".git",
+      ".idea",
+      ".github",
+      ".vscode"
+    ],
+    "release-type": "dotnet",
+    "include-component-in-tag": false,
+    "include-v-in-tag": true,
+    "prerelease": true,
```

Comment:
Do you know if we need to include the pre-release type property as well?

Reply:
I believe this config option is not supposed to be true based on what it does from the documentation.

> _Create the GitHub release as prerelease. Defaults to `false`._

Reply:
thanks for confirming!